.github/workflows/docker-publish-release.yaml

```diff
@@ -22,11 +37,7 @@ jobs:
       workload-id-provider: projects/1094498259535/locations/global/workloadIdentityPools/gh-op-geth-release/providers/github-by-repos
       service-account: op-geth-release@blockchaintestsglobaltestnet.iam.gserviceaccount.com
       artifact-registry: us-west1-docker.pkg.dev/devopsre/celo-blockchain-public/op-geth
-      tags: ${{(
-        contains(github.event.release.tag_name, 'rc') || 
-        contains(github.event.release.tag_name, 'beta') || 
-        contains(github.event.release.tag_name, 'alpha') 
-        && github.event.release.tag_name || format('{0},stable,latest', github.event.release.tag_name)
-        ) }}
+      platforms: linux/amd64,linux/arm64
+      tags: ${{ needs.Set-Tags.outputs.tags }}
```

Comment:
```suggestion
      tags: ${{ needs.set-tags.outputs.tags }}
```

Reply:
It's the job name = id, but I think it's not case sensitive. Just important not to confuse `Set-Tags` (note the `-`) with `set_tags` (with a `_`, that's a step id).